src/Calculator/Views/TitleBar.xaml

```diff
@@ -3,28 +3,53 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:converters="using:CalculatorApp.Converters"
              mc:Ignorable="d">
+    <UserControl.Resources>
+        <converters:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
+        <converters:BooleanToVisibilityNegationConverter x:Key="BooleanToVisibilityNegationConverter"/>
+    </UserControl.Resources>
     <Grid x:Name="LayoutRoot"
           Height="32"
           HorizontalAlignment="Stretch">
-        <VisualStateManager.VisualStateGroups>
-            <VisualStateGroup x:Name="WindowFocusStates">
-                <VisualState x:Name="WindowFocused"/>
-                <VisualState x:Name="WindowNotFocused">
-                    <VisualState.Setters>
-                        <Setter Target="AppName.Foreground" Value="{ThemeResource SystemControlForegroundChromeDisabledLowBrush}"/>
-                    </VisualState.Setters>
-                </VisualState>
-            </VisualStateGroup>
-        </VisualStateManager.VisualStateGroups>
-        <TextBlock x:Name="AppName"
+        <Grid x:Name="BackgroundElement"
+              Background="Transparent"
+              Height="32">
+            <VisualStateManager.VisualStateGroups>
+                <VisualStateGroup x:Name="WindowFocusStates">
+                    <VisualState x:Name="WindowFocused"/>
+                    <VisualState x:Name="WindowNotFocused">
+                        <VisualState.Setters>
+                            <Setter Target="AppName.Foreground" Value="{ThemeResource SystemControlForegroundChromeDisabledLowBrush}"/>
+                        </VisualState.Setters>
+                    </VisualState>
+                </VisualStateGroup>
+            </VisualStateManager.VisualStateGroups>
+            <TextBlock x:Name="AppName"
                    x:Uid="AppName"
                    Margin="12,0,12,0"
                    HorizontalAlignment="Left"
                    VerticalAlignment="Center"
                    Foreground="{ThemeResource TitleBarForegroundBaseHighBrush}"
                    FontSize="12"
                    TextAlignment="Left"
-                   TextTrimming="CharacterEllipsis"/>
+                   TextTrimming="CharacterEllipsis"
+                   Visibility="{x:Bind Model.CalculatorViewModel.IsAlwaysOnTop, Converter={StaticResource BooleanToVisibilityNegationConverter}, Mode=OneWay}"/>
+        </Grid>
+        <Button x:Name="AlwaysOnTopButton" Grid.Row="0"
+                FontFamily="{StaticResource CalculatorFontFamily}"
+                Style="{ThemeResource CommandBarFlyoutEllipsisButtonStyle}"
+                Content="&#xEE49;"
+                Click="AlwaysOnTopButtonClick"
```

Comment:
NIT: `AlwaysOnTopButtonClicked` or `AlwaysOnTopButton_Click`

It looks like the first format is more used in the app.